src/Apps/Artwork/Components/OtherAuctions.tsx

```diff
@@ -1,6 +1,6 @@
 import { Box, Flex } from "@artsy/palette"
 import { OtherAuctions_sales } from "__generated__/OtherAuctions_sales.graphql"
-import { AuctionCardFragmentContainer as AuctionCard } from "Components/v2/AuctionCard"
+import { AuctionCardFragmentContainer as AuctionCard } from "Components/v2/AuctionCardLuxon"
```

Comment:
What do you think of updating the existing components to use Luxon vs creating largely duplicate Luxon variations? Might be easier to digest the diff.

Related to https://github.com/artsy/reaction/pull/2476#discussion_r288696089, it would be great to identify opportunities to delegate formatting / TZ manipulation to Metaphysics, where possible.

Also might be worthwhile to start with artsy/palette and work our way up through artsy/reaction to artsy/force.

Reply:
I was thinking at first to make a copy with it's own story so its super clear that everything is displaying the same as the original. After that's verified I can replace the original files.

Reply:
Sounds reasonable but I hope we also have tests to verify that the formatted dates/times look correct.